index.html

```diff
@@ -311,14 +316,19 @@ <h3>Normative Specifications</h3>
                     <dt id="integrity" class="spec">Linked Data Integrity (LDI)</dt>
                     <dd>
                         <p>
-                            This specification defines a generic framework for expressing proofs of integrity of Linked Datasets via, for example, digital signatures, proofs of work, or proofs of existence. Beyond the generic (normative) framework, this deliverable also provides a normative definition for Linked Data Signatures, as well as a means to express these proofs in an RDF Dataset in a way that enables a 3rd party to verify the integrity of the data.
+                            This specification defines a structure for embedding signatures in JSON-LD documents. The group will define that embedding mechanism to work with RDH, though the hashing algorithm will be identified as an assertion in the structure, allowing the same structure to be used with other hashing algorithms. This deliverable enables 3rd parties to verify that the data has not changed since it was signed. Time-permitting, this signature may be designed to express proofs of work or proofs of existance.
```

Comment:
Signatures can’t be expressed in other RDF serializations? This seems overly narrow.

Reply:
@gkellogg there is a branch mis-match... The original sentence figuring JSON-LD was there in the previous version of the charter proposal (which was a huge mistake:-), but is not there in the latest version, and @ericprud will have to update this PR...

Reply:
It was. I changed it to "in JSON-LD documents".

Because the name was less misleading, i updated PR #73 and am closing this one.